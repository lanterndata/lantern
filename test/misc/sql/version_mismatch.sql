-- Validate that creating an index with mismatched versions fails
CREATE INDEX ON sift_base1k USING lantern_hnsw (v) WITH (dim=128, M=4);
