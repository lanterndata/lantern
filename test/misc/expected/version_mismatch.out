-- Validate that creating an index with mismatched versions fails
CREATE INDEX ON sift_base1k USING lantern_hnsw (v) WITH (dim=128, M=4);
WARNING:  LanternDB binary version does not match the version in SQL. This can cause errors as the two APIs may differ. Please run `ALTER EXTENSION lantern UPDATE` and reconnect before attempting to work with indices
ERROR:  Attempting to build lantern index, but the SQL version and binary version do not match. This can cause errors. Please run `ALTER EXTENSION lantern UPDATE and reconnect
